src/index.js

```diff
@@ -6,12 +6,16 @@ import program from "commander";
 import pck from "./../package.json";
 
 program.version(pck.version)
-       .option('-u --username [username]', 'Your del.icio.us username')
```

Comment:
Oops, it must be `'-u --username username'` -- brackets defining its being optional. Initially missed by me.